docs/flutter/v2/guides/quickstart.mdx

```diff
@@ -5,316 +5,868 @@ nav: 2.0
 
-```dart section=MuteUnMuteAudio sectionIndex=1
+```dart
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
 
-/// audio unmute
-hmsSDK.switchAudio(isOn: true);
+  static Future<bool> getPermissions() async {
+    if (Platform.isIOS) return true;
+    await Permission.camera.request();
+    await Permission.microphone.request();
+    await Permission.bluetoothConnect.request();
 
-/// audio mute
-hmsSDK.switchAudio(isOn: false);
+    while ((await Permission.camera.isDenied)) {
+      await Permission.camera.request();
+    }
+    while ((await Permission.microphone.isDenied)) {
+      await Permission.microphone.request();
+    }
+    while ((await Permission.bluetoothConnect.isDenied)) {
+      await Permission.bluetoothConnect.request();
+    }
+    return true;
+  }
 ```
 
-### Mute/UnMute Video
+### Step 2: Implement join screen
+
+This section will help you create the join screen user interface. To keep it simple for the quickstart, we have not created many UI elements; you can refer to the [sample app implementation](https://github.com/100mslive/100ms-flutter/tree/main/example/lib/preview) for a complete Preview/Join user interface. 
+
+```dart
+// UI to render join screen
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        color: Colors.black,
+        child: Center(
+          child: ElevatedButton(
+            style: ButtonStyle(
+              shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+                RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(8.0),
+                ),
+              ),
+            ),
+            // Function to push to meeting page
+            onPressed: () async {
+              await getPermissions();
+              Navigator.push(
+                context,
+                CupertinoPageRoute(builder: (_) => const MeetingPage()),
+              );
+            },
+            child: const Padding(
+              padding: EdgeInsets.symmetric(vertical: 20, horizontal: 20),
+              child: Text(
+                'Join',
+                style: TextStyle(fontSize: 20),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+```
+
+### Step 3: Implement meeting page
+
+You can check the below snippet to create a widget as the user interface to show the video tiles of local and remote peers. `HMSUpdateListener` plays a significant role in rendering video or displaying any information regarding the room.
+
+100ms SDK provides callbacks to the client app about any change or update happening in the room after a user has joined by implementing `HMSUpdateListener`. 
+
+To join a room, you need to create an `HMSConfig` instance and use that instance to call the join method of `HMSSDK`.
```

Comment:
More information about the App Token used in HMSConfig should be added here. Token Generation is a common pitfall

Reply:
done